### ISSUE TITLE
Loader HTML or Callback

### ISSUE THREAD

**Issue by aogaili** [gauravchl/react-image-story#3]
@gauravchl  Would it be possible to pass a callback to signal when the rendering of the gallery is complete? Alternatively, we can pass loader HTML to the component. 

The issue is that when I upload an image to a third party storage (amazon s3), it takes a bit of time and previously image gallery used to show "preparing images" and then render when ready however with the new version it's showing nothing and there is no way to figure out state of the component so we can display loading indicator.

**Comment by aogaili** [comment]
I've added this feature locally by making the following modification to the imagesPrepared function, and passing onLoad cb to the component.

I think that would be a good add to smart gallery.

 ```
imagesPrepared() {
    if (!this.images || !this.images.length) return;
    const prepared = !this.images.some((img) => img.loading);
    if (prepared) {
      this.props.onLoad && this.props.onLoad();
    }
    return prepared;
  },
```

**Comment by gauravchl** [comment]
@aliogaili Cool! new props 'onLoad' is now available @0.1.2
https://github.com/gauravchl/react-smart-gallery/commit/22157b0f7df29bdd5582a120479a09d4f55ae79a

It will run after all the images(supplied initially) rendered in UI.

**Comment by aogaili** [comment]
Yup, works great thanks @gauravchl !
